demo/addons/godot-openxr/runtimes.json

```diff
@@ -0,0 +1,14 @@
+[
+    {
+        "name": "SteamVR",
+        "path": "C:\\Program Files (x86)\\Steam\\steamapps\\common\\SteamVR\\steamxr_win64.json"
+    },
+    {
+        "name": "Oculus",
+        "path": "C:\\Program Files\\Oculus\\Support\\oculus-runtime\\oculus_openxr_64.json"
+    },
+    {
+        "name": "Microsoft MR",
```

Comment:
I've added Microsoft to the list even though we don't support it right now due to lack of DirectX support. Still someone might want to be able to change it back to MS before leaving Godot.

Reply:
Switching back isn't an issue anymore but I would still leave it in because why not.